situp-plugins/elasticsearch/src/main/java/com/amazon/situp/plugins/sink/elasticsearch/RequestFunction.java

```diff
@@ -0,0 +1,13 @@
+package com.amazon.situp.plugins.sink.elasticsearch;
+
+/**
+ * Represents a function that accepts one argument and produces a result or throws Exception.
```

Comment:
Why not reuse the java Function itself?

Reply:
This is b/c java Function `apply` method does not throw Exception, which `client::bulk` does not fit into

Reply:
Got it, take a look at this [comment](https://github.com/opendistro-for-elasticsearch/simple-ingest-transformation-utility-pipeline/pull/33#discussion_r492139914), you might not even the Function.